README.md

```diff
@@ -97,12 +117,21 @@ If you're looking for how to download, build, install and run Helios, keep readi
 Prerequisities
 --------------
 
+The binary release of Helios is built for Ubuntu 14.04.1 LTS, but Helios should
```

Comment:
Do we actually specifically build for Ubuntu 14.04 or is it just a generic deb package? Is it built from jdeb plugin specified in pom.xml? I'm not too familiar with this part.